scripts/cluster/create_multinode_cluster.sh

```diff
@@ -32,7 +34,9 @@ Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-tim
 EOF'
 sudo systemctl daemon-reload
 
-sudo kubeadm init --ignore-preflight-errors=all --cri-socket /run/containerd/containerd.sock --pod-network-cidr=192.168.0.0/16
+sudo kubeadm init --v=7 --apiserver-advertise-address=10.0.1.1 --cri-socket /run/containerd/containerd.sock --pod-network-cidr=10.168.0.0/16
```

Comment:
@HongyuHe @Lightxyz what these arguments are and what was the reason for the change?